Documentation/AdministratorManual/Index.rst

```diff
@@ -39,22 +36,22 @@ Edit file :file:`.htaccess` (or your virtual host) to read:
 .. _admin-manual-nginx:
 
 Installation (Nginx)
---------------------
+====================
 
 Edit your `server` block to read:
 
 .. code-block:: nginx
 
    location / {
-       rewrite ^/fileadmin/(?!(_processed_/)) /index.php last;
```

Comment:
NO, that's Apache configuration, official documents indents by 4: https://httpd.apache.org/docs/2.4/sections.html

Reply:
You manual - totally your decision. For docteam maintained manuals we've decided to give readability on small devices like cellphones highest priority and we are using three blank for one indentation level whenever the language allows it.